RELEASE

```diff
@@ -1,9 +1,38 @@
-    ___                _____            __    
-   /   |  ____  ____  / ___/_________  / /___ 
+    ___                _____            __
+   /   |  ____  ____  / ___/_________  / /___
   / /| | / __ \/ __ \ \__ \/ ___/ __ `/ // _ \
  / ___ |/ /_/ / /_/ /___/ / /__/ /_/ / //  __/
-/_/  |_/ .___/ .___//____/\___/\__,_/_/ \___/ 
-      /_/   /_/                               
+/_/  |_/ .___/ .___//____/\___/\__,_/_/ \___/
+      /_/   /_/
+AppScale version 3.3.0, released June, 2017
+Highlights of features/bugs in this release:
+- Added support for Ubuntu Xenial
+- Improved autoscaling mechanism to rely on resources capacity
+- Removed unnecessary db request while adding task
+- Fixed a bug that caused a DB request to hang indefinitely
+- Improved log collection command on appscale-tools
+- Improved script for enabling datastore-viewer
+- Fixed a bug with installation on Azure (eJabberd certificate)
+- Manual scaling with add-instances in cloud
+- Added retry for queue operations to increase reliability
+- Fail loudly if AppScale is not configured
```

Comment:
Maybe different wording here... "Clearer error messages if AppScale is not configured correctly"